buildSrc/src/main/java/org/elasticsearch/gradle/testclusters/ElasticsearchNode.java

```diff
@@ -1127,9 +1165,38 @@ private void createConfiguration() {
         } catch (IOException e) {
             throw new UncheckedIOException("Could not write config file: " + configFile, e);
         }
+
+        tweakJvmOptions(configFileRoot);
         LOGGER.info("Written config file:{} for {}", configFile, this);
     }
 
+    private void tweakJvmOptions(Path configFileRoot) {
```

Comment:
Is this additional config required for some reason?

Reply:
By default the jvm options write into the specific distro folder. If we say we don't rely on shared distro directories and keep using hard links and copy as a fallback we can remove it again. Otherwise we need this to keep the distro folders clean.